server/Dockerfile

```diff
@@ -0,0 +1,18 @@
+
+FROM ruby:2.6.5
+
+
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends nodejs \
+    && apt-get install -y --no-install-recommends libpq-dev
+
+
+
+WORKDIR /usr/src/app
```

Comment:
you should be using `/app` instead of `/usr/src/app`.  While `/usr/src` is used for storing source code in a Linux environment, as far as containers are concerned, it's much more common to see `/app` used to host the application source code.  It's become essentially a de-facto standard